app/src/main/java/com/nilhcem/blenamebadge/ui/fragments/base/BaseFragmentViewModel.kt

```diff
@@ -0,0 +1,112 @@
+package com.nilhcem.blenamebadge.ui.fragments.base
+
+import android.graphics.drawable.Drawable
```

Comment:
No android imports in the view model.

Reply:
Fixed them Sir, have a look

Reply:
I have a doubt though, what If I want to do processing on bitmaps or drawables and then assign it to a view. According to the pattern I should always work on my business logic in the DAO classes while it also does not allow you to pass the drawable or bitmap through the viewmodel, any suggestions would be appreciated sir :)

Reply:
You can do it in fragment itself, or can create a function in utils.

Reply:
@liveHarshit Alright, Let me improve the pr, give me 5 more minutes

Reply:
@liveHarshit there was a potential context and uri leak in MainActivityViewHolder as well, Thanks for your advice, Ive improved the implementation. Please have a look